coveralls.json

```diff
@@ -0,0 +1,12 @@
+{
+  "coverage_options": {
+    "minimum_coverage": 100
```

Comment:
Yes you're right I think I just use coveralls to produce a report then codecov is using this report to check the coverage. But I need to check again the flow